### DISCUSSION TITLE
Время-пространство или Пространство-время?

### DISCUSSION THREAD

**Discussion by tigran123** [tigran123/ud#1] [Идеи и вопросы]
В английском оригинале Пятого Эпохального Откровения используется очень странная форма "time-space" вместо "space-time", т.е. время на первом месте. Возникает вопрос: переводить буквально как "времени-пространственный" или стандартно как "пространственно-временной"? В данный момент я выбрал второй вариант, но отметил этот факт сноской в [0:1.9](https://github.com/tigran123/ud/blob/bd1e03625350cca88b1fb0cdd2dadfe6f9eebbda/tex/p000.tex#L18) 

Но в тех местах, где время и пространство разделены, я перевожу в правильном порядке, например "вселенные времени и пространства".

Будет интересно узнать, какие мнения имеются на этот счёт у других участников проекта.

PS. Восемь лет назад я написал статью о восприятии пространства и времени животными, людьми и сверхлюдьми. Мне кажется, что прочтение этой статьи было бы полезно при обсуждении данного вопрос, так что вот ссылка [http://www.bibles.org.uk/articles/T.Aivazian-The-Higher-Dimensions-of-Space.pdf](http://www.bibles.org.uk/articles/T.Aivazian-The-Higher-Dimensions-of-Space.pdf)

**Comment by irinachernov** [comment]
Возможно, действительно важно сохранить порядок слов в форме: "времени-пространственный", или возможно: "время-пространственный", раз в тексте Откровения это выражение употребляется, и неоднократно, именно в таком порядке.

**Reply by tigran123** [reply]
Эта фраза встречается в 22 параграфах по всему Предисловию, т.е. такое изменение означает перезапись аудио версии...

**Reply by irinachernov** [reply]
Хорошо, давай перепишем.

**Comment by alyuev** [comment]
Поиск по книгам прошлого (за 1900-1930 года) особо результатов не дал. Вот пару раз встретилось: 

1. https://books.google.com.ua/books?id=v70tAAAAMAAJ&q=%22time-space%22&dq=%22time-space%22&hl=ru&sa=X&ved=2ahUKEwiXodHhwYfuAhWNxosKHWqjA_oQ6AEwAnoECAYQAg
2. https://books.google.com.ua/books?id=vjpMAAAAMAAJ&q=%22time-space%22&dq=%22time-space%22&hl=ru&sa=X&ved=2ahUKEwiXodHhwYfuAhWNxosKHWqjA_oQ6AEwCXoECAUQAg
3. https://books.google.com.ua/books?id=5kIIAQAAIAAJ&pg=PA690&dq=%22time-space%22&hl=ru&sa=X&ved=2ahUKEwiXodHhwYfuAhWNxosKHWqjA_oQ6AEwAHoECAIQAg#v=onepage&q=%22time-space%22&f=false

**Reply by tigran123** [reply]
Спасибо, очень интересно, особенно книга Schlick -- она уже 4 года в нашей библиотеке, но только сейчас,  благодаря твоей находке, я решил обратить на неё внимание.

**Reply by tigran123** [reply]
Ещё один пример "time-space" -- книга "Эволюция физики" Эйнштейна и Инфельда, опубликованная в 1938 г. В русском переводе заменено обычной формой "пространство-время", как и следовало ожидать. Между прочим, в Откровении использованы печатные источники вплоть до середины 1940-х гг, так что 1938 г. это не такая уж "поздняя" дата для источника.

**Comment by tigran123** [comment]
Вообще-то это оффтопик, но я не могу удержаться -- вот, нашёл "объяснение" разницы между "space-time" и "time-space" [https://www.quora.com/What-is-the-difference-between-space-time-and-time-space](https://www.quora.com/What-is-the-difference-between-space-time-and-time-space) -- от физика, доктора наук из Гарварда, "из современных" (дата поста 2018), в которой он пытается внушить всем, на основе своего [не]понимания процедуры квантования в КТП, что, якобы пространство и время, как были разделены до Минковского, так и остались. Какой абсурд и совершенное непонимание сути теории относительности... Самое комичное это объяснение указанной выше "разницы" -- "разница заключается в порядке слов" ГЕНИАЛЬНО! :)

**Comment by tigran123** [comment]
Сказано -- сделано: https://github.com/tigran123/ud/commit/3dfdac822dc336b9c99acc51584666ee501e6196

**Reply by irinachernov** [reply]
Как хорошо!
